src/main.js

```diff
@@ -1,16 +1,30 @@
-import Vue from 'vue'
-// @ts-ignore
-import App from './App.vue'
-import BootstrapVue from 'bootstrap-vue';
+import Vue from "vue";
+import VueRouter from "vue-router";
```

Comment:
Normally, we will have a folder `src/router/index.js` to sperate code of router, and for vuex, we use `src/store/index.js`.